planning/costmap_generator/test/test_points_to_costmap.cpp

```diff
@@ -0,0 +1,215 @@
+// Copyright 2022 The Autoware Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+#include <rclcpp/rclcpp.hpp>
+
+#include <gtest/gtest.h>
+
+using pointcloud = pcl::PointCloud<pcl::PointXYZ>;
+class PointsToCostmapTest : public ::testing::Test
+{
+protected:
+  void SetUp() override { rclcpp::init(0, nullptr); }
+
+  ~PointsToCostmapTest() override { rclcpp::shutdown(); }
+
+  grid_map::GridMap constrcut_gridmap();
+
+public:
+  double grid_resolution_ = 1;
+  double grid_length_x_ = 20;
+  double grid_length_y_ = 20;
+  double grid_position_x_ = 0;
+  double grid_position_y_ = 0;
+};
+
+grid_map::GridMap PointsToCostmapTest::constrcut_gridmap()
+{
+  grid_map::GridMap gm;
+
+  // set gridmap size
+  gm.setFrameId("map");
+  gm.setGeometry(
+    grid_map::Length(grid_length_x_, grid_length_y_), grid_resolution_,
+    grid_map::Position(grid_position_x_, grid_position_y_));
+
+  // set initial value
+  gm.add("points", 0);
+
+  // set car postion in map frame to center of grid
+  grid_map::Position p;
+  p.x() = 0;
+  p.y() = 0;
+  gm.setPosition(p);
+
+  return gm;
+}
+
+// gridy
+// |  mapx-------
+// |            |
+// |            |
+// |            |
+// |            mapy
+// |__________________gridx
```

Comment:
By the way, are there any problems if I add `_` to each name?

```suggestion
// grid_y
// |  map_x-------
// |            |
// |            |
// |            |
// |            map_y
// |__________________grid_x
```